website/content/en/docs/scorecard-alpha/_index.md

```diff
@@ -0,0 +1,222 @@
+| Spec Block Exists | This test checks the Custom Resource(s) created in the cluster to make sure that all CRs have a spec block. | basic-check-spec-test |
+
+### OLM Integration
+
+| Test        | Description   | Short Name |
+| --------    | -------- | -------- |
+| OLM Bundle Validation | This test validates the OLM bundle manifests found in the bundle that is passed into scorecard.  If the bundle contents contain errors, then the test result output will include the validator log as well as error messages from the validation library.  See this [document][olm-bundle] for details on OLM bundles.| olm-bundle-validation-test |
+| Provided APIs have validation |This test verifies that the CRDs for the provided CRs contain a validation section and that there is validation for each spec and status field detected in the CR. | olm-crds-have-validation |
+| Owned CRDs Have Resources Listed | This test makes sure that the CRDs for each CR provided via the `cr-manifest` option have a `resources` subsection in the [`owned` CRDs section][owned-crds] of the CSV. If the test detects used resources that are not listed in the resources section, it will list them in the suggestions at the end of the test. | olm-crds-have-resources |
+| Spec Fields With Descriptors | This test verifies that every field in the Custom Resources' spec sections have a corresponding descriptor listed in the CSV.| olm-spec-descriptors-test |
+| Status Fields With Descriptors | This test verifies that every field in the Custom Resources' status sections have a corresponding descriptor listed in the CSV.| olm-status-descriptors-test |
+
+## Scorecard Output
+
+### JSON format
+
+Here is an example of the JSON format produced by a scorecard test:
+
+```json
+{
+  "metadata": {
+    "creationTimestamp": null
+  },
+  "log": "",
+  "results": [
+    {
+      "name": "basic-check-spec",
+      "description": "Custom Resource has a Spec Block",
+      "state": "pass"
+    }
+  ]
+}
+```
+
+### Text format
+
+Here is an example of the text format produced by a scorecard test:
+
+```
+	basic-check-spec                    : pass
+	CR: 
+	Labels: 
+```
+
+This output is structured according to the golang struct found
+[here][scorecard-struct].
+
+## Exit Status
+
+The scorecard return code is 1 if any of the tests executed did not 
+pass and 0 if all selected tests pass.
+
+## Extending the Scorecard with Custom Tests
+
+Scorecard can execute custom tests provided by ISVs or end users if 
+the custom tests following some mandated conventions including:
+
+ * tests are implemented within a container image
+ * tests accept an entrypoint which include a command and arguments
+ * tests produce v1alpha2 scorecard output in JSON format with no extraneous logging in the test output
+ * tests can obtain the bundle contents at a shared mount point of /bundle
+ * tests can access the Kube API using an in-cluster client connection
+
+See here for an example of a custom test image written in golang.  
```

Comment:
missing the example.

Reply:
May would be better add the example code here instead of having link unless it is auto-gen in some part of the code.

Reply:
once we include our custom image into the repo (another PR), I'll update this to point to that example.

Reply:
It is fine. Can we leave it commented with a todo?

Reply:
I added a TODO so we wont' forget.